### ISSUE TITLE
[Feature Request] Re-enter draw mode on open wall

### ISSUE THREAD

**Issue by Moonlight63** [s-leger/archipack#210]
I couldn't find where feature requests should be made, so I am posting it here. I would like to know if it is possible to continue drawing walls (and anything else that is drawable) after clicking away and the modal disappears. I find myself constantly misclicking when drawing walls, and I either have to delete the wall and start over or use the manipulator arrows to add segments. Why can't you just continue to click and draw walls just like you do the first time you create a wall, as long as the wall you want to add on to is selected in the outliner and manipulators are active? I think it would be much more intuitive if selecting a wall had the option to "enter draw mode" to bring the modal back up and continue drawing. This should only work if the wall is open of course. Would this be possible to add?

**Comment by s-leger** [comment]
Hi,
This feature is avaliable in archipack pro only.

**Comment by Moonlight63** [comment]
OH, yes, I am talking about archipack pro. I have it. How do I do this then? It seems like all of the documentation and tutorials are for the built in free version. Is there anywhere I can go to learn how to use the pro features?

**Comment by s-leger** [comment]
Select the wall, hold ALT and press "Wall" wall button again restart the draw tool.

**Comment by Moonlight63** [comment]
Ok, I tried this, and it does not work. I have Archipack Pro 2.3.1. I have looked into the code to try to debug why, and assuming that this functionality comes from archipack_preset.py line 456, this only works on linking windows and doors. I don't see anything that extends this functionality to walls. When I hold ALT and click 'Wall' with a wall currently selected, the preset window comes up just like normal and creates a new wall.
